packages/load-test/helpers.ts

```diff
@@ -0,0 +1,189 @@
+import { sleep } from '../../common'
+import { expect } from 'chai'
+import { deployContract } from 'ethereum-waffle'
+import L2DepositTracker = require('../../contracts/build-ovm/L2DepositTracker.json')
+import L1DepositInitiator = require('../../contracts/build/L1DepositInitiator.json')
+import L2TxStorage = require('../../contracts/build-ovm/L2TxStorage.json')
+import { Contract } from '@ethersproject/contracts'
+
+export const deployLoadTestContracts = async (
+  l1Wallet,
+  l2Wallet,
+  L2_DEPOSIT_TRACKER_ADDRESS?,
+  L1_DEPOSIT_INITIATOR_ADDRESS?,
+  L2_TX_STORAGE_ADDRESS?
+) => {
+  let l2DepositTracker
+  let l1DepositInitiator
+  let l2TxStorage
+  console.log('connected to L2 wallet at:', l2Wallet.address)
+  console.log('connected to L1 wallet at:', l1Wallet.address)
+  if (!L2_DEPOSIT_TRACKER_ADDRESS) {
+    l2DepositTracker = await deployContract(l2Wallet, L2DepositTracker)
+    console.log('l2DepositTracker address on L2:', l2DepositTracker.address)
```

Comment:
Do we need to `console.log` these things?

Reply:
moved to use the `core-utils` logger - it's useful when running these load tests against a live testnet so you can know the contract addresses you're connected to